e2e/cypress/integration/api/iam/02_policies_api.spec.ts

```diff
@@ -87,11 +110,359 @@ describeIfIAMV2('policies API', () => {
+      }).then((response) => {
+          nonAdminToken = response.body.token.value;
+      });
+
+      cy.request({
+        auth: { bearer: adminToken },
+        method: 'POST',
+        url: '/apis/iam/v2beta/policies',
+        body: {
+            name: 'token-access',
+            id: `${cypressPrefix}-token-access-policy-${now}`,
+            members: [ `token:${nonAdminTokenID}` ],
+            statements: [
+              {
+                effect: 'ALLOW',
+                actions: [ 'iam:policies:*' ],
+                projects: [ '*' ]
+              },
+              {
+                effect: 'ALLOW',
+                actions: [ 'iam:projects:assign' ],
+                projects: [ project1.id ]
+              }
+            ],
+            projects: []
+        }
+      });
+    });
+
+    after(() => {
+      cy.cleanupProjectsByIDPrefix(adminToken, cypressPrefix);
+      cy.cleanupPoliciesByIDPrefix(adminToken, cypressPrefix);
+      cy.cleanupTokensByIDPrefix(adminToken, cypressPrefix);
+    });
+
+    beforeEach(() => {
+      cy.request({
+        auth: { bearer: adminToken },
+        method: 'DELETE',
+        url: `/apis/iam/v2beta/policies/${policyID}`,
+        failOnStatusCode: false
+      });
+    });
+
+    afterEach(() => {
+      cy.request({
+        auth: { bearer: adminToken },
+        method: 'DELETE',
+        url: `/apis/iam/v2beta/policies/${policyID}`,
+        failOnStatusCode: false
+      });
+    });
+
+    describe('POST /apis/iam/v2beta/policies', () => {
+      it('admin can create a new policy with no projects', () => {
+        cy.request({
+            auth: { bearer: adminToken },
+            method: 'POST',
+            url: '/apis/iam/v2beta/policies',
+            body: policyWithProjects(policyID, [], statementProjects)
+        }).then((response) => {
+            expect(response.body.policy.projects).to.have.length(0);
+        });
+      });
+
+      it('admin can create a new policy with multiple projects', () => {
+        cy.request({
+            auth: { bearer: adminToken },
+            method: 'POST',
+            url: '/apis/iam/v2beta/policies',
+            body: policyWithProjects(policyID, [project1.id, project2.id], statementProjects)
+        }).then((response) => {
+            expect(response.body.policy.projects).to.have.length(2);
+        });
+      });
+
+      it('admin gets a 404 when it attempts to create ' +
+         'a policy with a project that does not exist', () => {
+        cy.request({
+            auth: { bearer: adminToken },
+            method: 'POST',
+            url: '/apis/iam/v2beta/policies',
+            failOnStatusCode: false,
+            body: policyWithProjects(policyID,
+              [project1.id, project2.id, 'notfound'], statementProjects)
+        }).then((response) => {
+            expect(response.status).to.equal(404);
+        });
+      });
+
+      it('non-admin with project1 assignment access can create a new policy ' +
```

Comment:
"project1" would not be meaningful for someone reviewing test results--they would only have the test names. Suggest remove "project1" here and in other tests below, perhaps "non-admin with a specific project assignment access..."